playbooks/openshift-node/scaleup.yml

```diff
@@ -34,8 +34,6 @@
 
 - name: install nodes
   hosts: new_nodes
-  vars:
```

Comment:
bootstrap endpoint can be set as a fact for new_nodes.
```
- set_fact:
    openshift_bootstrap_endpoint: "{{ openshift_bootstrap_uri if openshift_bootstrap_uri is defined else 'https://' + openshift_install_config['metadata']['name'] + '-api.' + openshift_install_config['baseDomain'] + ':49500/config/worker' }}"
```

Reply:
Done.... will set for all nodes with master endpoints being overridden in associated plays when building new clusters. We can revamp that later if necessary.

Reply:
Latest version handles BYOR control plane, nothing left to handle later.